### PR TITLE
tools/upip.py Fix changed PyPi URL (issue #274).

### PR DESCRIPTION
**Pull request by peterhinch**
Fixes fail caused by changed PyPi URL https://github.com/micropython/micropython-lib/issues/274

### CODE REVIEW

File: ports/unix/modules/upip.py [hunk 1]
```diff
@@ -1 +0,0 @@
-../../../tools/upip.py
```
Comment (dpgeorge) [inline ports/unix/modules/upip.py]: Is this removal intentional?

Comment (peterhinch) [reply ports/unix/modules/upip.py]: Sorry, no.

**Comment by peterhinch** [comment]
PR https://github.com/micropython/micropython-lib/pull/275 is a more complete solution so I'll close this.
